src/index.ts

```diff
@@ -202,3 +176,35 @@ function replaceAllWhitespacesImpl6(string: string): string {
 
   return bufferTo.toString('utf-8', 0, index);
 }
+
+function isWhiteSpaceCharCode(charCode: number): boolean {
```

Comment:
# before

```
  fast-impl-7:
    783 ops/s, ±0.25%   | fastest

  normalize-space-x:
    384 ops/s, ±0.16%   | slowest, 50.96% slower
```

# after

```
  fast-impl-7:
    1 000 ops/s, ±0.28%   | fastest

  normalize-space-x:
    379 ops/s, ±0.14%     | slowest, 62.1% slower
```

Reply:
yes, all for it, that was roughly the idea)

Reply:
how about using set ? `return whiteSpaceCodeSet.has(charCode)`

Reply:
aha, so set is actually slow! very interesting :)

Reply:
pushed boolean array implementation =)